README.md

```diff
@@ -112,18 +115,20 @@ NativeAudio.preloadSimple({
 /**
  * Platform: Android/iOS
  * This method will load more optimized audio files for background into memory.
- * @param assetPath - relative path of the file or absolute url (http://)
+ * @param assetPath - relative path of the file or absolute url (file://)
```

Comment:
Good catch. I was in the impression that I could use this plugin with external resources too. 
The preloadSimple comment still contains "http://" though. 

Maybe even add a remark to the readme that this plugin only works with local files?

Reply:
sure, I can do that. 

actually I'm pretty sure that before this PR, you could only load assets from the app bundle / resources, as I tried to use `file` URLs and failed, and thus added the `isUrl` parameter to get them to work (but only for `preloadComplex`).

My use case is downloading audio files to the app cache directory and playing them from there, hence I added support for `file` URLs.

Reply:
I added that second commit, let me know what else I can do.

Reply:
Looks good to me 👍  Now we wait for someone to approve this PR 😉

Reply:
sounds good!